draft-ietf-ppm-dap.md

```diff
@@ -168,6 +173,9 @@ aggregator.
 - Bump draft-irtf-cfrg-vdaf-13 to 14 {{!VDAF}} and adopt changes to the
   ping-pong API (\#705, \#718).
 
+- Represent the Time and Duration types as the number of time_precision
+  intervals since the UNIX epoch, rather than seconds (\*) (#720).
```

Comment:
This isn't quite right: a Duration isn't measured relative to any epoch or any starting point at all. A Duration relative to a starting point is an Interval.